src/components/PineTypes/Boolean.js

```diff
@@ -20,6 +21,25 @@ export const rules = {
 
 export const validate = isBoolean
 
+export const normalize = value => {
+  if (isString(value)) {
+    switch (value.toLowerCase()) {
+    case 'true':
```

Comment:
So, does prettier like aligning the `case`s right below the `switch`!?!?!? :alien: :satellite: